### PR TITLE
Add frozen string literal

### PR DESCRIPTION
**Pull request by MittchoBaroco**
The test seems to be broken on master, but I tried to generate a component from my local gem and it still works.

also I didn't know if it was useful to comment to the module template `/lib/generators/component/templates/rb.erb.`

#64

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ Style/MethodCallWithoutArgsParentheses:
 Style/MethodDefParentheses:
   Enabled: true
 
+Style/FrozenStringLiteralComment:
+  Enabled: true
```
Comment (florentferry) [inline .rubocop.yml]: `EnforcedStyle: always`

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline Gemfile]: Add an extra empty line.

File: Rakefile [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline Rakefile]: Same here.

File: features/support/setup.rb [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline features/support/setup.rb]: Same here.

File: komponent.gemspec [hunk 5]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline komponent.gemspec]: Same here.

File: lib/komponent/railtie.rb [hunk 6]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline lib/komponent/railtie.rb]: Same here.

File: test/komponent/komponent_helper_test.rb [hunk 7]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline test/komponent/komponent_helper_test.rb]: Same here.

File: test/test_helper.rb [hunk 8]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (florentferry) [inline test/test_helper.rb]: Same here.

**Comment by MittchoBaroco** [comment]
Sounds good 👌🏻 I will do the changes shortly

**Pull request closed without merging**